### PR TITLE
fix: Edit "You've been nominated" text (#4641)

### PR DESCRIPTION
**Pull request by richsalz**
Mention that the schedule is posted to ietf-announce and give a link to subscribe to that.  Use vague wording to handle all cases.

### CODE REVIEW

**Comment by richsalz** [comment]
Sorry for the noise; forced push/rebase etc.  Learning how to submit PR's here.

File: ietf/dbtemplate/fixtures/nomcom_templates.xml
Comment (rjsparks) [inline ietf/dbtemplate/fixtures/nomcom_templates.xml:1]: This file is essentially legacy - it was used to populate the database long ago, and is not used by e.g. the tests. 

Changes here will not affect production.
What you really want is a migration that changes a row in the database. I can show you how to build that.

File: ietf/nomcom/migrations/0001_initial.py
Comment (rjsparks) [inline ietf/nomcom/migrations/0001_initial.py:1]: This migration has long ago been run. Changing it here has no effect on production. Please revert this file in your branch.

File: ietf/dbtemplate/fixtures/nomcom_templates.xml
Comment (rjsparks) [reply ietf/dbtemplate/fixtures/nomcom_templates.xml:1]: After we create the migration you want, (or edit the template in the database directly), you'll need to revert this file in your branch

File: ietf/nomcom/models.py [hunk 1]
```diff
@@ -48,7 +48,7 @@ class NomCom(models.Model):
 
     group = ForeignKey(Group)
     send_questionnaire = models.BooleanField(verbose_name='Send questionnaires automatically', default=False,
-                                             help_text='If you check this box, questionnaires are sent automatically after nominations.')
+                                             help_text='If you check this box, questionnaires are sent automatically after nominations. DO NOT CHECK if they are not ready yet.')
```
Comment (rjsparks) [inline ietf/nomcom/models.py:51]: This is fine, but it also needs a new companion migration.

**Review by rjsparks — CHANGES_REQUESTED** [review]

File: ietf/nomcom/migrations/0001_initial.py
Comment (rjsparks) [reply ietf/nomcom/migrations/0001_initial.py:1]: (again, hold until we create the migration you really wanted so we can translate your suggestion into the migration.)

File: ietf/nomcom/migrations/0001_initial.py [hunk 2]
```diff
@@ -50,7 +50,7 @@ class Migration(migrations.Migration):
             fields=[
                 ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                 ('public_key', models.FileField(blank=True, null=True, storage=ietf.utils.storage.NoLocationMigrationFileSystemStorage(location=None), upload_to=ietf.nomcom.models.upload_path_handler)),
-                ('send_questionnaire', models.BooleanField(default=False, help_text='If you check this box, questionnaires are sent automatically after nominations.', verbose_name='Send questionnaires automatically')),
+                ('send_questionnaire', models.BooleanField(default=False, help_text='If you check this box, questionnaires are sent automatically after nominations. DO NOT CHECK if they are not ready yet.', verbose_name='Send questionnaires automatically')),
```
Comment (jennifer-richards) [inline ietf/nomcom/migrations/0001_initial.py:53]: Should add a new migration instead of modifying this one

**Review by rjsparks — APPROVED** [review]

**Comment by rjsparks** [comment]
holding to bring in after the postgres transition is complete

**Comment by richsalz** [comment]
I think this is ready for review now.

**Pull request merged**